libpod/runtime_ctr.go

```diff
@@ -47,6 +47,32 @@ func (r *Runtime) NewContainer(ctx context.Context, rSpec *spec.Spec, options ..
 	return r.newContainer(ctx, rSpec, options...)
 }
 
+func (r *Runtime) PrepareVolumeOnCreateContainer(ctx context.Context, ctr *Container) (retErr error) {
+	// Copy the content from the underlying image into the newly created
+	// volume if configured to do so.
+	if !r.config.Containers.PrepareVolumeOnCreate {
+		return nil
+	}
+
+	defer func() {
+		if retErr != nil {
+			if err := ctr.cleanupStorage(); err != nil {
+				logrus.Errorf("error cleaning up container storage %s: %v", ctr.ID(), err)
+			}
+		}
+	}()
+
+	mountPoint, err := ctr.mountStorage()
+	if err == nil {
+		// Finish up mountStorage
+		ctr.state.Mounted = true
+		ctr.state.Mountpoint = mountPoint
+		logrus.Infof("Container %s storage %s mounted successfully", ctr.Name(), mountPoint)
```

Comment:
This should be Debug, not Info.

Reply:
Sure, next commit will take care of that.